# Define the goal
string arm_name
geometry_msgs/Pose pose
float64 depth
float64 closed_angle
float64 open_angle


---
# Define the result
ToolPose pose
string info
---
# Define a feedback message
ToolPose pose
string info

